investigator/overlays/test/imagestreamtag.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: image.openshift.io/v1
 kind: ImageStream
 metadata:
-  name: unresolved-package-handler
+  name: investigator
 spec:
   tags:
   - name: latest
     from:
       kind: DockerImage
-      name: quay.io/thoth-station/unresolved-package-handler:v0.1.2
+      name: quay.io/thoth-station/investigator:v0.1.2
```

Comment:
we have not released it, would the next release be v0.2.0 ?

Reply:
we can do a minor release yes after https://github.com/thoth-station/investigator/pull/48 is merged